sdjournal/read.go

```diff
@@ -38,7 +38,10 @@ type JournalReaderConfig struct {
 	// the array is empty, entries will not be filtered.
 	Matches []Match
 
-	MaxEntryLength uint64 // the max allowable length for an entry (default is 64K)
+	// explicitly set MaxEntryLenght to 0, which means unlimited
+	DisableMaxEntryLength bool
+	// the max allowable length for an entry (default is 64K)
+	MaxEntryLength uint64
```

Comment:
Since this value was hitherto not actually used, you don't have to worry about it being backwards-compatible, so let's just give it the same behaviour as the systemd threshold: 0 means unlimited.